packages/components/src/components/Alert/Alert.tsx

```diff
@@ -194,6 +162,52 @@ export const Alert: FC<AlertProps> = ({
     lineHeight: 30,
   }
 
+  const _header = () => {
+    const headerText = header ? (
+      <Text style={headerFont}>{header}</Text>
+    ) : description ? (
```

Comment:
As noted in general comment, Q&A with Jessica noted we don't need to support the no-header case for expandable Alerts so this and subsequent logic can be adjusted.

Reply:
Removed this logic